pkg/operation/botanist/component/kubeapiserver/configmaps.go

```diff
@@ -79,6 +85,89 @@ func admissionPluginsConfigFilename(name string) string {
 	return strings.ToLower(name) + ".yaml"
 }
 
+func computeRelevantAdmissionPluginRawConfig(plugin AdmissionPluginConfig) ([]byte, error) {
+	var (
+		nothingToMutate    = (plugin.Config == nil || plugin.Config.Raw == nil) && len(plugin.Kubeconfig) == 0
+		mustDefaultConfig  = (plugin.Config == nil || plugin.Config.Raw == nil) && len(plugin.Kubeconfig) > 0
+		kubeconfigFilePath = volumeMountPathAdmissionKubeconfigSecrets + "/" + admissionPluginsKubeconfigFilename(plugin.Name)
+	)
+
+	if len(plugin.Kubeconfig) == 0 {
+		// This makes sure that the path to the kubeconfig is overwritten if specified in case no kubeconfig was
+		// provided. It prevents that users can access arbitrary files in the kube-apiserver pods and disguise them as
+		// kubeconfigs for their admission plugin configs.
```

Comment:
Just one question, What did you mean by this comment?
How can users disguise arbitrary files as kubeconfigs?